letsencrypt/tests/cli_test.py

```diff
@@ -43,7 +43,11 @@ def _call(self, args):
         return ret, stdout, stderr, client
 
     def test_no_flags(self):
-        self.assertRaises(SystemExit, self._call, [])
+        with mock.patch('letsencrypt.cli.run') as mock_run:
+            from letsencrypt import cli
+            cli.VERBS["run"] = mock_run
+            self._call([])
+            self.assertEqual(1, mock_run.call_count)
```

Comment:
backref to regression from this PR: https://github.com/letsencrypt/letsencrypt/pull/863#discussion_r40856338